mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksExtensions.kt

```diff
@@ -246,6 +246,31 @@ fun <V : Any> args() = object : ReadOnlyProperty<Fragment, V> {
     }
 }
 
+/**
+ * Fragment argument delegate that makes it possible to set fragment args without
+ * creating a key for each one.
+ *
+ * To create nullable arguments, define a property in your fragment like:
+ *     `private val listingId by argsOrNull<MyArgs?>()`
+ *
+ * Each fragment can only have a single argument with the key [Mavericks.KEY_ARG]
+ */
+fun <V : Any> argsOrNull() = object : ReadOnlyProperty<Fragment, V?> {
```

Comment:
```suggestion
fun <V> argsOrNull() = object : ReadOnlyProperty<Fragment, V?> {
```
I _think_ you can safely emit the type narrowing since the return type is nullable.